release.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script will check the state of the main branch of flux-lsp for
+# conditions that would allow for a release to occur. If those conditions
+# are met, a signed tag is created and *pushed to github* where the CI
+# will take over and publish the extension.
+#
+# WARNING: This script will *push directly to master*. Please make sure
+# you understand the contents of this script and the consequences of its
+# execution before running it.
+set -e
+
+if [[ ! $(command -v hub) ]]; then
+	echo "Please install the hub tool and re-run."
+	exit 1
+fi
+
+if [[ ! $(command -v jq) ]]; then
+	echo "Please install the jq tool and re-run."
+	exit 1
+fi
+
+if [[ ! $(cargo bump -v) ]]; then
+    echo "Please install cargo bump and re-run"
+    exit 1
+fi
+
+TEMPDIR=$(mktemp -d -t flux-release.XXXX)
+echo "Using fresh install in $TEMPDIR"
+cd $TEMPDIR
+git clone git@github.com:influxdata/flux-lsp.git > /dev/null 2>&1
+cd $TEMPDIR/flux-lsp
+
+cargo bump patch --git-tag
```

Comment:
Do we ever anticipate the need to do something besides a patch version in the near future?